bin/i18n/i18n_script_utils.rb

```diff
@@ -44,22 +44,16 @@ def self.with_synchronous_stdout
   #      is only managed programmatically, we avoid wrapping to make the git
   #      diffs smaller and change detection easier.
   #
-  #   2. Quote 'y' and 'n'. Psych intentionally departs from the YAML spec for
-  #      these strings: https://github.com/ruby/psych/blob/8e880f7837db9ed66032a1dddc85444a1514a1e3/test/psych/test_boolean.rb#L21-L35
-  #      But Crowdin sticks strictly to the YAML spec, so here we add special
-  #      logic to ensure that we conform to the spec when outputting for Crowdin
-  #      consumption.
-  #      See https://github.com/gvvaughan/lyaml/issues/8#issuecomment-123132430
+  #   2. Make every line uses the Double-quote format so it is consistent between
+  #      syncs and also the values will be on one line. If we don't tell Psych to
+  #      use Double-quotes, then it might use the Block format which is multi-line.
   def self.to_crowdin_yaml(data)
     ast = Psych.parse_stream(Psych.dump(data))
 
-    # Make sure we treat the strings 'y' and 'n' as strings, and not bools
```

Comment:
Our sync-in has a loop which would looks for `y` or `n` values in the YAML files and adds single quotes around them. This PR changes the loop to instead add double-quotes around all values, rather than just `y` or `n`.